### ISSUE TITLE
Versus X Quora

### ISSUE THREAD

**Issue by samcarecho** [RafaelReinehr/versus#1]
Reinventing the wheel is not productive, therefore I must ask.
What're the differences between Versus and Quora?

Ref: quora.com

**Comment by RafaelReinehr** [comment]
May i answer in portuguese, because part of the answer is already done, and if anyone need a translation ill do i asap.

Ambas plataformas são bem diferentes. O Yahoo e o quora basicamente permite que uma resposta geral seja escolhida ou produzida e qualificada.

O stack overflow e similares permite que as melhores respostas se sobressaiam.

O versus permite pesar todas as respostas, pros, contras e áreas cinzas. E permite ao usuário decidir por si o que é melhor, já que nem sempre o que eu estou buscando não é o que é melhor para maioria.

Com acessos a todos estes dados, uso meu bem senso e decido com a maioria ou então por mim mesmo. Autonomia aliada à inteligência coletiva.

Acho que alguns bons exemplos de uso do versus é o da busca de soluções para um dilema ético ou para escolha de diferentes opções viáveis para uma mesma pergunta,

Exemplo: 

Se eu empresto meu carro para um amigo levar meu filho na escola e na volta ele sofre um acidente e gera um prejuízo de 3 mil reais, de quem deve ser o ônus do pagamento do conserto.

BOX 1 > Do proprietário do veículo, pois o amigo estava lhe fazendo um favor.
BOX 2 > Do amigo motorista, pois era habilitado e enquanto dirigia era o responsável pelo veículo
BOX 3 > Deve haver um acerto solidário entre ambos de acordo com a capacidade econômica de cada um
BOX 4 > outra opção possível, etc.

A imagem anexa no google docs (e que tentei postar na wiki explica melhor).

No exemplo dado lá, o versus serviria para comparar diferentes tipos de tratamentos possíveis para uma condição patológica. E serviria para reforçar os prós e contras de cada um, levantando as contingências em que um seria benéfico e vantajoso em relação ao outro e vice-versa.

O Quora, Yahoo e Satck Overflow não tem esta potência analítica e organizativa, penso eu.

**Comment by iuriguilherme** [comment]
## Translation of what @RafaelReinehr didn't feel like translating (rightfully)

Both tools are very different. Yahoo and Quora basically allows a given answer to be chosen, produced and rated.

Stack Overflow and similar software allows the best answers to stay on top of all the others.

Versus allow to weight every answer, the pros, the cons and gray areas. Also, it allows the user to decide for her/himself¹ what works better for her/him¹, since it is not always that what I am looking for is that which is better for most people.

With access to all this data, I use my good sense and decide along with the majority, or by myself. Autonomy linked with collective intelligence.

I think some good examples of Versus use is to search for solutions for an ethical dilemma or for the choice between different viable options for the same question.

Example:

If I borrow my car to a friend to take my child at school and on the way back he suffers an accident, and as a consequence of that generates a R$ 3.000 loss, who's responsible for paying that debt?
- BOX 1 > The owner of the vehicle, for the friend driver was doing her/him¹ a favor.
- BOX 2 > The friend driver, because he had permission to drive and also was responsible for the vehicle while driving.
- BOX 3 > A solidarity agreement shall happen between both, taking into account each one's budget.
- BOX 4 > Another possible solution, etc.

The attached image on Google Docs (and which I have tried to post on the wiki) explains better.²

In the example given _(on the image, I presume)_, Versus would serve to compare different kinds of possible treatment for a given pathological disease. And it would serve to reinforce the pros and cons of each one, rising the contingencies in which one would be beneficial and advantageous over the other and vice versa.

Quora, Yahoo and Stack Overflow does not have this analytical and organizative power, I presume.

---

_¹: I privilege the feminine in brazilian portuguese, I'm not sure if that's how it is done in english, this was the first attempt._
_²: @RafaelReinehr is talking about some image that I think is the one he sent me sometime ago, not sure where it is publicly available though. If it is the same I am thinking of, I could upload here._

**Comment by iuriguilherme** [comment]
From their [about page](https://www.quora.com/about):

> Quora's mission is to share and grow the world's knowledge. We want to democratize access to knowledge of all kinds

First fail. Quora didn't had an API (and probably still doesn't have), effectively [forcing people to write all kinds of scrapers and pseudo APIs](https://github.com/search?q=quora&type=Repositories) to be able to get the questions and answers from the site.

Any tool is pretty much useless in the sense of sharing and growing knowledge and democratize access when its source code is closed and it doesn't export all the data, or huge portions of it, in an easy way. Even to be able to export all data that belongs to an user or individual questions is unacceptable.

> Quora makes it easy to get your questions answered, share your own knowledge, and browse the most interesting information people across the world want to share.

This is a double fail, actually. The only part they've really delivered is the "easy to browse", maybe, I haven't bothered to try, I didn't agree with the Google/Facebook/Twitter/Quora terms of service.

---

So in a way this is actually **not** reinventing the wheel. From a libre software's perspective, we are still not aware that there is such thing as a wheel, as we don't know how their car moves around to begin with.

Given the premise that we want to allow people to share and broaden knowledge, there is not even discussion on whether the software must be open sourced or not.

Some people with good hearth are leading the way and starting to write [open source quora clones](https://github.com/search?q=quora.com&type=Repositories) using Rails. Then we could compare the differences.

---

@RafaelReinehr explanations are still valid for open source tools like [AskBot](https://askbot.com/), though. Even if Quora gets open sourced, they're still valid for Quora as well. The paradigm is totally different.

**Comment by RafaelReinehr** [comment]
You are right, @iuriguilherme, about the image.  I wasnt able to upload the picture... Its the same i shared with you.
